packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -60,6 +61,8 @@ export function useListLayout<T>(state: ListState<T>, density: ListViewProps<T>[
         estimatedRowHeight: ROW_HEIGHTS[density][scale],
         padding: 0,
         collator
+        // TODO: fix loader height, may need to adjust the list layout
+        // loader calc to properly position the loader
```

Comment:
Note: the placement of the loader and its height will need to be adjusted since it doesn't quite work out of the box with the ListLayout (ListLayout won't have the loader take all available vertical height if a `loaderHeight` is specified. Omitting the `loaderHeight` causes the loadingMore spinner size to match the visible rect.) May need to extend the ListLayout to change the loader calculations or change ListLayout loader calculations as a whole.

Perhaps use the same calculations as CardView: https://github.com/adobe/react-spectrum/blob/main/packages/@react-spectrum/card/src/GridLayout.tsx#L204-L218. Out of scope for this docs PR though